cartography/intel/gcp/iam.py

```diff
@@ -705,6 +709,57 @@ def attach_role_to_domain(
     )
 
 
+def _set_used_state_tx(
+    tx: neo4j.Transaction, project_id: str, common_job_parameters: Dict, update_tag: int,
+) -> None:
+    ingest_role_used = """
+    MATCH (n:GCPRole)<-[:RESOURCE]-(:GCPProject{id: {GCP_PROJECT_ID}})
+    <-[:OWNER]-(:CloudanixWorkspace{id: {WORKSPACE_ID}})
+    WHERE (n)<-[:ASSUME_ROLE]-() AND n.lastupdated = {update_tag}
```

Comment:
can you please point me to the code where we determine the ASSUME_ROLE relationship for all the clouds?